.jshintrc

```diff
@@ -0,0 +1,24 @@
+{
+    "white": true,
+    "trailing": true,
```

Comment:
I think the `white` and `trailing` have been deprecated -- I can't find any mention them on the jshint docs: http://jshint.com/docs/options/ only traces of these options in jshint's repository.